lucien-ios/AppDelegate.swift

```diff
@@ -21,4 +22,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
+    private func application(application: UIApplication, openURL url: URL, options: [UIApplicationOpenURLOptionsKey: AnyObject]) -> Bool {
+        return GIDSignIn.sharedInstance().handle(url as URL!,
```

Comment:
The style guide is a little ambiguous on this one, but when calling a function or init, I prefer each parameter to be on its own line like so:
```
return GIDSignIn.sharedInstance().handle(
    url as URL!,
    sourceApplication: options[UIApplicationOpenURLOptionsKey.sourceApplication] as? String,
    annotation: options[UIApplicationOpenURLOptionsKey.annotation]
)
```
Also, I know it was in the code provided by Google, but you don't need the force cast to URL